src/test/java/org/jabref/model/entry/AuthorListTest.java

```diff
@@ -375,6 +374,16 @@ public void testCompanyAuthor() {
         assertEquals(expected, author);
     }
 
+    @Test
+    public void testCompanyAuthorAndPerson() {
+        Author author = AuthorList.parse("{JabRef Developers} and Stefan Kolb").getAuthor(0);
+        Author expected = new Author(null, null, null, "JabRef Developers", null);
+        assertEquals(expected, author);
+        author = AuthorList.parse("{JabRef Developers} and Stefan Kolb").getAuthor(1);
+        expected = new Author("Stefan", "S.", null, "Kolb", null);
+        assertEquals(expected, author);
```

Comment:
Why not simply
`assertEquals(Arrays.asList(new Author(JabRef), new Author(Kolb)), AuthorList.parse(".."))`?